res/res_fax.c

```diff
@@ -4631,7 +4631,7 @@ static int acf_faxopt_read(struct ast_channel *chan, const char *cmd, char *data
 	char *filenames;
 
 	if (!details) {
-		ast_log(LOG_ERROR, "channel '%s' can't read FAXOPT(%s) because it has never been written.\n", ast_channel_name(chan), data);
+		ast_debug(1, "channel '%s' can't read FAXOPT(%s) because it has never been written.\n", ast_channel_name(chan), data);
```

Comment:
I think this should be level 3. It's not really that much of a high priority debug message.

Reply:
Or more specifically, they should all be 3.